app/src/androidTest/java/com/stevesoltys/seedvault/e2e/screen/impl/DocumentPickerScreen.kt

```diff
@@ -0,0 +1,19 @@
+package com.stevesoltys.seedvault.e2e.screen.impl
+
+import android.widget.EditText
+import com.stevesoltys.seedvault.e2e.screen.UiDeviceScreen
+
+object DocumentPickerScreen : UiDeviceScreen<DocumentPickerScreen>() {
+
+    val createNewFolderButton = findObject { text("CREATE NEW FOLDER") }
+
+    val useThisFolderButton = findObject { text("USE THIS FOLDER") }
```

Comment:
ok these are specific to manage documents permission missing, so it won't work on my test phones which isn't a problem right now, but prevent me from running the tests outside of CI (because I can't run emulators).

Reply:
Ahh yeah. Sorry about that. One option is if we can somehow get the [generic system images](https://developer.android.com/topic/generic-system-image/releases) to boot in an emulator, we can test with the permission. I couldn't get this working though :disappointed:. If you know how let me know and we can switch over, it would be better without all the Google apps anyways.

Reply:
What we could do instead is *not* request the MANAGE_DOCUMENTS permission for instrumentation test (via a modified manifest). So every device would behave the same.